.github/workflows/sync-docs.yml

```diff
@@ -0,0 +1,71 @@
+env:
+    REPO_CONSOLIDATE: "devonfw-guide"
+    BUILD_USER: ${{ secrets.BUILD_USER }}
+    BUILD_USER_PASSWD: ${{ secrets.BUILD_USER_PASSWD }}
+    BUILD_USER_EMAIL:  ${{ secrets.BUILD_USER_EMAIL }}
+    EMAIL: "icsddevonfwsupport.apps2@capgemini.com"
```

Comment:
Please don't put the email here. Take it from the secrets like in other repositories

Reply:
Sure.. the BUILD_USER_EMAIL will be used.